src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -158,6 +158,12 @@
      */
     @Parameter(property = "docker.skip.extendedAuth", defaultValue = "false")
     protected boolean skipExtendedAuth;
+    
+    /** 
+     * Skip building tags
+     */
+    @Parameter(property = "docker.skip.tag", defaultValue = "false")
+    protected boolean skipTag;
```

Comment:
Can we move this to the Mojos only where it makes sense (Build/Remove/Push) ? This leads to some duplication, true, but makes it clear that its not usable in other Mojos.